### PR TITLE
Added tire pressure and engine oil

### PR DESCRIPTION
**Pull request by theodorewahle**

### CODE REVIEW

File: src/main/java/com/smartcar/sdk/data/VehicleOil.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  private double lifeRemaining;
+
+  /**
+   * Initializes a new instance of VehicleOil
+   *
+   * @param lifeRemaining the engine oil's remaining life span (as a percentage).
+   */
+  public VehicleOil(double lifeRemaining) {
+    this.lifeRemaining = lifeRemaining;
+    this.lifeRemaining = lifeRemaining;
```
[the remark this answers is not in the document]
Comment (theodorewahle) [reply src/main/java/com/smartcar/sdk/data/VehicleOil.java]: oh jeez. great catch. i will fix this.

[the remark this answers is not in the document]
Comment (theodorewahle) [reply src/main/java/com/smartcar/sdk/data/VehicleOil.java]: fixed

File: src/integration/java/com/smartcar/sdk/IntegrationTest.java [hunk 2]
```diff
@@ -38,6 +38,8 @@
             "read_location",
             "read_odometer",
             "read_fuel",
+            "read_tires",
+            "read_engine_oil",
```
Comment (adolfoportilla) [inline src/integration/java/com/smartcar/sdk/IntegrationTest.java]: missing tests for this 2 functions scopes, you can add the tests on the VehicleTest.java folder

Comment (theodorewahle) [reply src/integration/java/com/smartcar/sdk/IntegrationTest.java]: i added these

File: src/integration/java/com/smartcar/sdk/VehicleTest.java [hunk 3]
```diff
@@ -105,6 +105,23 @@ public void testLocation() throws SmartcarException {
...
+
+     /**
+     * Tests that the vehicle's tire pressure can be obtained.
+     */
+    @Test(groups = "vehicle")
+    public void testTirePressure() throws SmartcarException {
+        SmartcarResponse response = this.vehicle.tirePressure();
+    }
+
+
```
Comment (adolfoportilla) [inline src/integration/java/com/smartcar/sdk/VehicleTest.java]: nit: extra line

Comment (theodorewahle) [reply src/integration/java/com/smartcar/sdk/VehicleTest.java]: fixed! thank u for catching this

**Pull request merged**